packages/context/src/lib/context-root.ts

```diff
@@ -94,10 +94,10 @@ export class ContextRoot {
       return;
     }
 
-    // Note, it's important to use the initial target via composedPath()
+    // Note, it's important to use the initial target
     // since that's the requesting element and the event may be re-targeted
     // to an outer host element.
-    const element = event.composedPath()[0] as HTMLElement;
+    const element = event.contextTarget as HTMLElement;
```

Comment:
To be interoperable with the context protocol we need to fallback to `composedPath()[0]` when `contextTarget` isn't set.

```suggestion
    const element = (event.contextTarget ?? event. composedPath()[0]) as HTMLElement;
```